### PR TITLE
Add caption and changed image

### PR DESCRIPTION
**Pull request by junyucao**
<h2>Creating pull request in order to merge change with master. Will likely need review in case something went wrong. 🏃 </h2>

1. My
2. Name
3. Is
4. Not

Task Remaining:
- [ ] get name
- [ ] make sure it's taken
- [ ] insert as 5

### CODE REVIEW

File: _posts/0000-01-05-junyucao.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+me when blanking out in meetings
```
Comment (githubteacher) [inline _posts/0000-01-05-junyucao.md:6]: What about adding an emoji: 💼 ?

**Review by githubteacher — CHANGES_REQUESTED** [review]
Hey, what about adding an emoji w/single line comment.

**Review by fahadun — DISMISSED** [review]
👎

**Review by Quintanillaa — COMMENTED** [review]
You failed.

File: _posts/0000-01-05-junyucao.md (hunk 1 above)
Comment (junyucao) [reply _posts/0000-01-05-junyucao.md:6]: but why?!?!

**Review by Quintanillaa — APPROVED** [review]
👍

**Review by githubteacher — APPROVED** [review]

**Pull request merged**